_posts/newsletters/2022-11-04-newsletter.md

```diff
@@ -0,0 +1,261 @@
+* [Platform for Advanced Scientific Computing (PASC23)](https://pasc23.pasc-conference.org),
+Davos, Switzerland, 26-28 June 2023
+   * Submission deadline: 11 Dec 2022 (no extensions) 
+    * Includes a track for research software engineering submissions
+    
+
+<a name="reads"></a>
+# **Interesting Reads, Videos or Podcasts**
+
+Items you may have missed on the blog and Slack:
+* Chase Million: [My Reading List for Research Software Engineers](https://bssw.io/blog_posts/my-reading-list-for-research-software-engineers), BSSw blog post
+* Videos of our virtual [US-RSE workshop](https://us-rse.org/virtual-workshop-2022/) in September 2022 are here
+    * [First Day](https://www.youtube.com/watch?v=Lf4KR0X7Gz0&ab_channel=USResearchSoftwareEngineerAssociation)
+    * [Second Day](https://www.youtube.com/watch?v=7Byp6BS32oM&ab_channel=USResearchSoftwareEngineerAssociation)
+* Matthew Greenhouse: [The James Webb Space Telescope Mission](https://www.youtube.com/watch?v=b3hDUhqW8ls&ab_channel=ScienceGatewaysCommunity), Video of keynote at [Gateways 2022](https://sciencegateways.org/gateways2022)
+* RSECon2022 took place in Newcastle, UK, in September 2022. Here are some of the recordings. More will become available soon.
+    * [Oracle keynote on Gravitational Waves](https://www.youtube.com/watch?v=0P_yAqhbZVA)
+    * [SocRSE AGM](https://www.youtube.com/watch?v=TLdTmtQ_VNI)
+    * [Keynote from Neil Ferguson](https://www.youtube.com/watch?v=u-4fXU0G6Ig)
+    * [RSE Funding –  eCSE Panel](https://www.youtube.com/watch?v=Db7F9_aVr5U)
+    * [Christopher Woods on Funding Software Maintenance](https://www.youtube.com/watch?v=JuCBQH7pcxU)
+ * Daniel S. Katz: [How groups can immediately benefit when new members join](https://danielskatzblog.wordpress.com/2022/10/31/new-group-members/) (blog post)
+
+
+<a name="elev-wg"></a>
+# **Elevating RSEs Working Group**
+
+Many of us have similar stories in regards to recognizing the criticality of our roles in supporting modern research endeavors, yet struggle to gain the recognition of that amongst campus leadership, which oftentimes still thinks in terms of more traditional university roles such as faculty and students.  The aim of the Elevating RSEs working group is to help all of us make this clearer at our various organizations.  A key aspect of this is making it very clear to researchers, as well as funding agencies, the scientific discoveries that we are enabling, the scholarship that could not be done otherwise.  As a first step we are soliciting from the RSE community stories of such discoveries that were RSE enabled!  If you have such a story please fill out this brief form [here](https://docs.google.com/forms/d/1zCf4CShRvK3ec1IJasTgGi0_a568gPVRZPMlv3nmlZY).  From these stories we will be doing three things in the short term:
```

Comment:
```suggestion
Many of us have similar stories in regards to recognizing the criticality of our roles in supporting modern research endeavors, yet struggle to gain the recognition of that amongst campus leadership, which oftentimes still thinks in terms of more traditional university roles such as faculty and students.  The aim of the Elevating RSEs working group is to help all of us make this clearer at our various organizations.  A key aspect of this is making it very clear to researchers, as well as funding agencies, the scientific discoveries that we are enabling, the scholarship that could not be done otherwise.  As a first step we are soliciting from the RSE community stories of such discoveries that were RSE enabled!  If you have such a story please fill out [this brief form](https://docs.google.com/forms/d/1zCf4CShRvK3ec1IJasTgGi0_a568gPVRZPMlv3nmlZY).  From these stories we will be doing three things in the short term:
```